spec/lib/diaspora_federation/validators/comment_validator_spec.rb

```diff
@@ -0,0 +1,34 @@
+module DiasporaFederation
+  describe Validators::CommentValidator do
+    let(:entity) { :comment_entity }
+
+    it_behaves_like "a common validator"
+
+    it_behaves_like "a diaspora id validator" do
+      let(:property) { :diaspora_id }
+      let(:mandatory) { true }
+    end
+
+    %i(guid parent_guid).each do |prop|
+      it_behaves_like "a guid validator" do
+        let(:property) { prop }
+      end
+    end
+
+    context "#author_signature and #parent_author_signature" do
```

Comment:
1. please use `describe` for properties and methods
2. you can switch the `describe` and the `each`, see: https://github.com/SuperTux88/diaspora_federation/blob/salmon/spec/lib/diaspora_federation/validators/profile_validator_spec.rb#L16-L23, then you have a `describe` for each property.